check_endpoints.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+BASE_URL="https://mainnet.aeternity.io/mdw/"
+
+cat README.md | grep -q "$REGEX"
```

Comment:
what you think about a default `REGEX`? would it supposed to be the `BASE_URL`?

Reply:
was from debugging